app/models/report_generators/data_quality/fy2017/q3.rb

```diff
@@ -52,7 +54,7 @@ def columns
         project_name: she_t[:project_name],
         RelationshipToHoH: e_t[:RelationshipToHoH],
         household_id: she_t[:household_id],
-        CoCCode: ec_t[:CoCCode],
+        CoCCode: e_t[:enrollment_coc],
```

Comment:
So this is an interesting situation.  In all likelihood, you can no longer run a 2017 DQ report (I'm not 100% sure though) and viewing an old one should show you the data as it was at the time.  However, if you _can_ run one, it should run with the logic from 2017.

Reply:
got it, I'll revert this. I guess if we keep the old code functional we don't need to migrate legacy data.